routes/index.js

```diff
@@ -50,42 +53,59 @@ router.post('/api/users', function (req, res) {
     console.log('Input field empty')
     return res.status(400).send('Input field empty')
   }
-
   // Checks for duplicate
-  const checkdup = {
-    text: 'SELECT COUNT(email) FROM users WHERE email = $1',
-    values: [email.toLowerCase()]
-  }
-  db.query(checkdup)
+  CheckDup(email)
     .then(result => {
-      if (result.rows[0].count !== 0) {
-        res.status(240)
+      if (result != 0) {
+        // console.log(result)
+        res.status(409).send({message:"exist"})
+      } else {
+
+        // Insert user data
+        const query = {
+          text: 'INSERT INTO users(firstname, lastname, email, secret) VALUES ($1, $2, $3, $4)',
+          values: [firstname.toLowerCase(), lastname.toLowerCase(), email.toLowerCase(), secret]
+        }
+        db.query(query)
+          .then(result => {
+            res.send([firstname, lastname, email, secret])
```

Comment:
As I said in `client/src/AddFriend`, don't just return raw SQL -- format it so that's easy to just use by the client. 
```
res.send({
  firstname: <>
  lastname: <>
  email: <>
  secret: <>
})
```